script.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
+
+// import { Modal } from "modal.js"
+
+window.onload = function () {
+  //Add Tags
+  addTagsClickHandler();
+
+  //showing and hiding of mobile screens
+  showAndHidePhoneDisplay();
+
+  //screen slider
+  slideScreensWithPhones();
+};
+
+const addTagsClickHandler = () => {
+  document.querySelector(".portfolio__tags").addEventListener("click", (e) => {
+    if (e.target.getAttribute("class") === 'tag') {
+      let clickedTag = e.target;
+      removeSelectedTags();
+      selectClickdeTag(clickedTag);
+       changeImageOrder()
+    }
+  });
+};
+
+const removeSelectedTags = () => {
+  let tags = document.querySelectorAll(".tag");
+  tags.forEach((tag) => {
+    tag.classList.remove("tag_selected");
+  });
+};
+
+const selectClickdeTag = (clickedTag) => {
+  clickedTag.classList.add("tag_selected");
+};
+
+//------------changing of the image's order
+const changeImageOrder = () => {
+  let layout = document.querySelector("#portfolio");
+  let images = document.querySelectorAll(".portfolio-picture");
+  //let n = Math.floor(Math.random() * 10);
+  images[0].remove();
+  layout.append(images[0]);
+};
+
+//-----------hide and show mobile displays//////////////
+const showAndHidePhoneDisplay = () => {};
+
+class ScreenAndButton {
```

Comment:
name... it doesn't explain the meaning of class

Reply:
deleted